vscode/l10n/bundle.l10n.en.json

```diff
@@ -16,13 +16,20 @@
   "jdk.downloader.label.detectedMachineArchitecture": "Detected Machine Architecture",
   "jdk.downloader.button.label.downloadAndInstall": "Install and start setup",
   "jdk.downloader.label.selectOpenJdkVersion": "Select Oracle OpenJDK Version",
+  "jdk.downloader.message.downloadProgressBar":"{jdkType} {jdkVersion} download in progress",
   "jdk.downloader.message.downloadingAndCompletingSetup": "Downloading and completing setup of {jdkType} {jdkVersion}...",
   "jdk.downloader.message.downloadCompleted":"{jdkType} {jdkVersion} for {osType} download completed!",
   "jdk.downloader.error_message.whileSavingFile": "Error while saving the file: {error}",
   "jdk.downloader.error_message.whileDownloading": "Error while downloading the : {jdkType} {error}",
   "jdk.downloader.message.downloadFailed": "{jdkType} {jdkVersion} for {osType} download failed, wrong checksum.",
   "jdk.downloader.error_message.downloadFailedHttpError": "HTTP Error {statusCode} - {statusMessage}",
-  "jdk.downloader.error_message.jdkExtractionError": "Error: {error}",
+  "jdk.downloader.error_message.anyError": "Error: {error}",
```

Comment:
instead of anyError lets provide Some context

Reply:
Actually I am not using this label anywhere, I have kept it since it was already present. So, should we remove this label or should I give a more generalized key like `jdk.extension.error_message.errorLabel`

Reply:
ok lets leave it as is then